pkg/odo/cli/pipelines/bootstrap.go

```diff
@@ -100,6 +100,7 @@ func NewCmdBootstrap(name, fullName string) *cobra.Command {
 
 	bootstrapCmd.Flags().StringVar(&o.GitOpsRepoURL, "gitops-repo-url", "", "GitOps repository e.g. https://github.com/organisation/repository")
 	bootstrapCmd.Flags().StringVar(&o.GitOpsWebhookSecret, "gitops-webhook-secret", "", "provide the GitHub webhook secret for GitOps repository (if not provided, it will be auto-generated)")
+	bootstrapCmd.Flags().BoolVar(&o.Overwrite, "overwrite", false, "allows to overwrite if there is an exixting gitops repository")
```

Comment:
s/exixting/existing/

Also, let's agree that our command help should start with an initial capital letter, some do, and some don't.